h2o-extensions/target-encoder/src/main/java/ai/h2o/targetencoding/TargetEncoderModel.java

```diff
@@ -33,9 +33,9 @@ public TargetEncoderModel(Key<TargetEncoderModel> selfKey, TargetEncoderParamete
 
   public static class TargetEncoderParameters extends Model.Parameters {
     public boolean _blending = false;
-    public BlendingParams _blending_parameters;
+    public BlendingParams _blending_parameters = TargetEncoder.DEFAULT_BLENDING_PARAMS;
```

Comment:
Added default values for Flow. Some defaults were actually missing. Luckily, Target Encoder defines defaults.